config/index.js

```diff
@@ -0,0 +1,8 @@
+const pkg = require('../package.json')
+
+const PORT = 5432
+const TEST_PORT = 5435
```

Comment:
`TEST_PORT` is currently not being used by any code and should therefore be deleted.